rdrf/rdrf/features/fh_multisection_tests.feature

```diff
@@ -0,0 +1,43 @@
+Feature: User operates on multisection items.
+  A user can:
+  Add items (single section data) to a multisection and delete an item
+  by checking the "Mark for Deletion" checkbox and saving.
+  
+  Background:
+    Given export "fh.zip"
+    Given a registry named "FH Registry"
+  
+  Scenario: Add multiple items to a multisection.
+    When I am logged in as curator
+    When I click Module "Main/Imaging" for patient "SMITH John" on patientlisting
+    Then location is "Main/Imaging"
+  
+    # Enter first item
+    And I enter value "01-01-2017" for form "Imaging" multisection "Carotid Ultrasonography" cde "Date" in item 1  
+    And I enter value "4.0" for form "Imaging" multisection "Carotid Ultrasonography" cde "Result (right)" in item 1
+    And I enter value "item 1" for form "Imaging" multisection "Carotid Ultrasonography" cde "Result" in item 1
+    When I upload2 file "/app/rdrf/rdrf/features/fh_file_upload.feature" for multisection "Carotid Ultrasonography" cde "Report" in item 1
```

Comment:
Apologies for the duplicate comment - I accidentally put this comment on the commit also.

Can we remove the path? In utils there is a "def utils_path():", that allows you to get the path to utils at runtime, perhaps pick a file relative to that location. The paths for stellar are constructed this way, for example.

It avoids having a hard coded path that will only work from within a docker container (admittedly, this is what we are doing 99.99% of the time)